corehq/apps/userreports/models.py

```diff
@@ -982,6 +982,14 @@ def map_col(column):
                 'columns': [x for x in (map_col(col) for col in self.columns) if x]
             }
 
+    @property
+    def report_type(self):
+        if self.location_column_id:
```

Comment:
is map_config effectively same as location_column_id?

Reply:
Yes. If `location_column_id` has a value, then `map_config` has a value, and if `location_column_id` doesn't have a value then `map_config` is `None`. (See [map_config method here](https://github.com/dimagi/commcare-hq/blob/443a0afd66edda3dd15e3fe8724cd8047d4e3425/corehq/apps/userreports/models.py#L975).) So checking `location_column_id` just saves a little compute.

Reply:
Also, this change is cherry-picked from here: https://github.com/dimagi/commcare-hq/pull/35931